var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -576,6 +576,10 @@ def cmake_args(self):
         if spec.satisfies("@1.10.8,1.13.0"):
             args.append(self.define("HDF5_INSTALL_CMAKE_DIR", "share/cmake/hdf5"))
 
+        # AOCC does not support _Float16
+        if spec.satisfies("@1.14.4: %aocc"):
+            args.append(self.define("HDF5_ENABLE_NONSTANDARD_FEATURE_FLOAT16", "OFF"))
```

Comment:
```suggestion
            args.append(self.define("HDF5_ENABLE_NONSTANDARD_FEATURE_FLOAT16", False))
```

minor, but adds a `:BOOLEAN` type otherwise it's `:STRING`